proposals/lambda-method-group-defaults.md

```diff
@@ -292,10 +373,10 @@ D1 d = Fun; // Allowed WITHOUT warning, since Fun and its default parameter are
 ### IL/Runtime Behavior
 
 The default parameter values will be emitted to metadata. The IL for this feature will be very similar in nature to the IL emitted for lambdas with `ref` and `out` parameters. A class which
-inherits from `System.Delegate` or similar will be generated, and the `Invoke` method will include `.param` directives to set default parameters -- 
-just as would be the case for a standard named delegate with default parameters.
+inherits from `System.Delegate` or similar will be generated, and the `Invoke` method will include `.param` directives to set default parameters or `System.ParamArrayAttribute` &ndash; 
+just as would be the case for a standard named delegate with default or `params` parameters.
 
-These delegate types are able to inspected at runtime, as normal.
+These delegate types are able to be inspected at runtime, as normal.
```

Comment:
>are able to be [](http://example.com/codeflow?start=21&length=14)

Perhaps "can be".

Reply:
Thanks. 0286e34e15bc1b1fb67c091c7364fa5db108f003